docs/source/package_reference/logging_methods.rst

```diff
@@ -1,37 +1,33 @@
 Logging methods
 ----------------------------------------------------
 
-🤗 Datasets tries to be very transparent and explicit about its inner working, but this can be quite verbose at times.
+🤗 Datasets strives to be transparent and explicit about how it works, but this can be quite verbose at times.We have included a series of logging methods which allow you to easily adjust the level of verbosity of the entire library. Currently the default verbosity of the library is set to ``WARNING``.
```

Comment:
nit:
```suggestion
🤗 Datasets strives to be transparent and explicit about how it works, but this can be quite verbose at times. We have included a series of logging methods which allow you to easily adjust the level of verbosity of the entire library. Currently the default verbosity of the library is set to ``WARNING``.
```